src/test/java/seedu/guest/storage/JsonAdaptedGuestTest.java

```diff
@@ -36,104 +36,104 @@ public class JsonAdaptedGuestTest {
             .collect(Collectors.toList());
 
     @Test
-    public void toModelType_validPersonDetails_returnsPerson() throws Exception {
-        JsonAdaptedGuest person = new JsonAdaptedGuest(BENSON);
-        assertEquals(BENSON, person.toModelType());
+    public void toModelType_validPersonDetails_returnsGuest() throws Exception {
```

Comment:
Should this be validGuestDetails rather than validPersonDetails in the method name?